### DISCUSSION TITLE
How can I write a hypergeometric function using typst?

### DISCUSSION THREAD

**Discussion by szdytom** [typst/typst#812] [Q&A] [answered]
what I wanted in LaTeX can be compose by:

```
$\mathrm F\left(\begin{gathered}a,b\\c\end{gathered}\middle|x\right)$
```

what I try in typst is:

```
$ upright(F)lr((mat(delim: #none, a_1comma dots comma a_m; b_1 comma dots comma b_m)|x)) $
```

the thing is all about the `\middle` directive. There seems no way to get `|` char longer in typst. it seems that typst should somehow add a `lmr` function.

**Comment by peng1999** [comment]
There is a ongoing PR #641 that implements want you want.

**Comment by szdytom** [comment, marked as answer]
do

```
$ upright(F)lr((lr(mat(delim: #none, a_1comma dots comma a_m; b_1 comma dots comma b_m)|)x)) $
```
